aiohttp/client_reqrep.py

```diff
@@ -147,7 +146,7 @@ def check(self, transport: asyncio.Transport) -> None:
 
 
 if ssl is not None:
-    SSL_ALLOWED_TYPES = (ssl.SSLContext, bool, Fingerprint, type(None))
```

Comment:
I think if we're going for backwards compatible first, then maybe just this line needs reverting?